__tests__/__system__/api/methods/install/Install.urimap.system.test.ts

```diff
@@ -0,0 +1,207 @@
+*                                                                                 *
+*/
+
+import { Session, selectProfileNameDesc } from "@zowe/imperative";
+import { ITestEnvironment } from "../../../../__src__/environment/doc/response/ITestEnvironment";
+import { TestEnvironment } from "../../../../__src__/environment/TestEnvironment";
+import { generateRandomAlphaNumericString } from "../../../../__src__/TestUtils";
+import { defineUrimapServer, defineUrimapClient, defineUrimapPipeline, deleteUrimap, disableUrimap, IURIMapParms, discardUrimap,
+         installUrimap, enableUrimap } from "../../../../../src";
+
+let testEnvironment: ITestEnvironment;
+let regionName: string;
+let csdGroup: string;
+let session: Session;
+
+function sleep(ms: number) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+const sleepTime = 2000;
+
+describe("CICS Install URImap", () => {
+
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            testName: "cics_cmci_install_urimap",
+            installPlugin: true,
+            tempProfileTypes: ["cics"]
+        });
+        csdGroup = testEnvironment.systemTestProperties.cmci.csdGroup;
+        regionName = testEnvironment.systemTestProperties.cmci.regionName;
+        const cmciProperties = await testEnvironment.systemTestProperties.cmci;
+
+        session = new Session({
+            user: cmciProperties.user,
+            password: cmciProperties.password,
+            hostname: cmciProperties.host,
+            port: cmciProperties.port,
+            type: "basic",
+            strictSSL: false,
+            protocol: testEnvironment.systemTestProperties.cmci.protocol as any || "http",
+        });
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+
+    const options: IURIMapParms = {} as any;
+
+    it("should install a URIMap of type server from CICS", async () => {
+        let error;
+        let response;
+
+        const urimapNameSuffixLength = 6;
+        const urimapName = "X" + generateRandomAlphaNumericString(urimapNameSuffixLength);
+
+        options.name = urimapName;
+        options.path = "fake";
+        options.host = "fake";
+        options.scheme = "http";
+        options.programName = "AAAA1234";
+        options.csdGroup = csdGroup;
+        options.regionName = regionName;
+        options.enable = false;
+        await defineUrimapServer(session, options);
+        await sleep(sleepTime);
+
+        try {
+            response = await installUrimap(session, options);
+        } catch (err) {
+            error = err;
+        }
+
+        expect(error).toBeFalsy();
+        expect(response).toBeTruthy();
+        expect(response.response.resultsummary.api_response1).toBe("1024");
+        await sleep(sleepTime);
+        await discardUrimap(session, options);
+        await sleep(sleepTime);
+        await deleteUrimap(session, options);
```

Comment:
If it helps in the future, feel free to place these cleanup actions in an `afterEach` hook. No worries on having multiple levels of `describe`s.
No change requested!